### ISSUE TITLE
Support NEXRAD Level III Super-Res

### ISSUE THREAD

**Issue by akrherz** [Unidata/gempak#72]
As of this week, "super-res" level III products are now flowing over NWS dissemination systems like NOAAPort.  This issue summaries work done to support these products in GEMPAK.  There are two relevant PNS/SCNs at play here:

- [PNS 21-55](https://www.weather.gov/media/notification/pdf2/pns21-55_reduce_sbn_wsr-88d.pdf) Soliciting Comments on a Change to NEXRAD Product Dissemination via NOAAPORT/SBN on December 1, 2021
- [SCN 21-96](https://www.weather.gov/media/notification/pdf2/scn21-96_sbn_super-res.pdf) Change to NEXRAD Product Dissemination via NOAAPort/SBNon or Around February 2, 2022

My understanding is that NCEP NAWIPS will **not** support these products as they are considered *enhancements* and national centers they support continue to migrate to AWIPS 2.  Having said that, never say never, but the GEMPAK community should try to see what can be done prior to anything "coming from on high".

So the pressing issue at hand here is if/when some of the pre-super-res products that are heavily used, like `N0Q` are removed from noaaport.  The expectation is that these products would still remain on other non-push dissemination systems, but the Unidata community is heavily tied to NOAAPort.  My understanding at the moment is that the products slated to be removed have not yet, due to various NWS national centers yet to migrate off noaaport for level III reception.

# Table 1 RADAR PRODUCTS TO BE REMOVED FROM NOAAPORT/SBN

WMO HEADING | PRODUCT|  DESCRIPTION
--- | --- | --
NXUS6i | GSM | General Status Message 2/GSM
SDUS7i | NET | Echo Tops 4km X 4km 41/ET
SDUS2i | N1S | Storm Relative Velocity .54nm X 1deg 16LVL 56/SRM 1.3-1.5DEG
SDUS2i | N2S | Storm Relative Velocity .54nm X 1deg 16LVL 56/SRM 2.4-2.5DEG
SDUS3i | N3S | Storm Relative Velocity .54nm X 1deg 16LVL 56/SRM 3.1-3.5DEG
SDUS4i | RCM | Radar Coded Message 74/RCM
SDUS4i | RSL | Archive III Status Product 152/ASP
SDUS8i | DOD | Dig. One Hour Difference Accumulation 174/DOD
SDUS8i | DSD | Dig. Storm Total Difference Accumulation 175/DSD
SDUS5i | NXQ | Base Reflectivity .54nm X 1deg 256LVL 94/DR -0.2DEG
SDUS5i | NYQ | Base Reflectivity .54nm X 1deg 256LVL 94/DR 0.0-0.2DEG
SDUS5i | NZQ | Base Reflectivity .54nm X 1deg 256LVL 94/DR 0.3-0.4DEG
SDUS5i | N0Q | Base Reflectivity .54nm X 1deg 256LVL 94/DR 0.5DEG
SDUS5i | NAQ | Base Reflectivity .54nm X 1deg 256LVL 94/DR 0.9DEG
SDUS2i | N1Q | Base Reflectivity .54nm X 1deg 256LVL 94/DR 1.3-1.5DEG
SDUS2i | NBQ | Base Reflectivity .54nm X 1deg 256LVL 94/DR 1.8DEG
SDUS2i | N2Q | Base Reflectivity .54nm X 1deg 256LVL 94/DR 2.4-2.5DEG
SDUS2i | N3Q | Base Reflectivity .54nm X 1deg 256LVL 94/DR 3.1-3.5DEG
SDUS5i | NXU | Base Velocity .13nm X 1deg 256LVL 99/DV -0.2DEG
SDUS5i | NYU | Base Velocity .13nm X 1deg 256LVL 99/DV 0.0-0.2DEG
SDUS5i | NZU | Base Velocity .13nm X 1deg 256LVL 99/DV 0.3-0.4DEG
SDUS5i | N0U | Base Velocity .13nm X 1deg 256LVL 99/DV 0.5DEG
SDUS5i | NAU | Base Velocity .13nm X 1deg 256LVL 99/DV 0.9DEG
SDUS2i | N1U | Base Velocity .13nm X 1deg 256LVL 99/DV 1.3-1.5DEG

# Table 2 RADAR PRODUCTS TO BE ADDED TO NOAAPORT/SBN
WMO HEADING | PRODUCT|  DESCRIPTION
--- | --- | --
SDUS5i | NXB | Base Reflectivity .13nm X .5deg 256LVL 153/SDR -0.2DEG
SDUS5i | NYB | Base Reflectivity .13nm X .5deg 256LVL 153/SDR 0.0-0.2DEG
SDUS5i | NZB | Base Reflectivity .13nm X .5deg 256LVL 153/SDR 0.3-0.4DEG
SDUS5i | N0B | Base Reflectivity .13nm X .5deg 256LVL 153/SDR 0.5DEG
SDUS5i | NAB | Base Reflectivity .13nm X .5deg 256LVL 153/SDR 0.9DEG
SDUS2i | N1B | Base Reflectivity .13nm X .5deg 256LVL 153/SDR 1.3-1.5DEG
SDUS2i | NBB | Base Reflectivity .13nm X 1deg 256LVL 153/SDR 1.8DEG
SDUS2i | N2B | Base Reflectivity .13nm X 1deg 256LVL 153/SDR 2.4-2.5DEG
SDUS2i | N3B | Base Reflectivity .13nm X 1deg 256LVL 153/SDR 3.1-3.5DEG
SDUS5i | NXG | Base Velocity .13nm X .5deg 256LVL 154/SDV -0.2DEG
SDUS5i | NYG | Base Velocity .13nm X .5deg 256LVL 154/SDV 0.0-0.2DEG
SDUS5i | NZG | Base Velocity .13nm X .5deg 256LVL 154/SDV 0.3-0.4DEG
SDUS5i | N0G | Base Velocity .13nm X .5deg 256LVL 154/SDV 0.5DEG 
SDUS5i | NAG | Base Velocity .13nm X .5deg 256LVL 154/SDV 0.9DEG
SDUS2i | N1G | Base Velocity .13nm X .5deg 256LVL 154/SDV 1.3-1.5DEG 

Hopefully we, the GEMPAK Community, can get this resolved quickly :)

**Comment by akrherz** [comment]
OK, initial attempt to load `N0B` in `nmap2`.  GUI allows selection of the `N0B` source, but then emits error messages of whilst attempting to load each file.

>  got a non-zero tbnids value          153           3

1. Added an entry in `gempak/tables/sat/imgtyp.tbl`, same error.
2. Added entry in `gempak/tables/rad/nidprd.tbl`, and it loads.

This is hopeful.

**Comment by akrherz** [comment]
Now trying `N0G` in `nmap2` and get a similar error message

>  got a non-zero tbnids value          154           3

Similar to the above, adding table entries allows this to load.  Hope springs eternal that this all may work without modifying code.

**Comment by akrherz** [comment]
`nex2img` appears up to the task of creating a `N0B` composite.
![n0b](https://user-images.githubusercontent.com/210858/156595720-64f694b8-117f-4416-96c8-9a9347c13a15.gif)

**Comment by sebenste** [comment]
One user I know would love to have this capability. If NCEP will not support it, and you have it working, would you consider making a 7.15.1 upgrade for this capability?

**Comment by akrherz** [comment]
@sebenste yes, I plan on making a release once #73 is merged.  In general, it is just some table updates.

**Comment by akrherz** [comment]
PR #73 merged, but am leaving this open for a bit more testing / review before declaring victory / making a GEMPAK patch release.
